app/views/event/participations/_list.html.haml

```diff
@@ -37,7 +37,7 @@
   - t.col(PhoneNumber.model_name.human(count: 2))                { |p| p.all_phone_numbers(cannot?(:show_details, p)) }
   - sortable_grouped_person_attr(t, address: false, zip_code: true, town: true) { |p| p.complete_address }
 
-  = render_extensions :list, locals: { table: t }
+  - event_participation_table_options(t, @event)
```

Comment:
ich denke das problem ist, dass `= render_extensions` verwendet wird. mit einem minus sollte es gehen:
```haml
- render_extensions :list, locals: { table: t }
```

dann braucht es die changes in den einzelnen wagons nicht und nur einen 1-liner im core

Reply:
Das fixed das Problem auch ja, war auch mein erster quick Fix in der PR History, nach Absprache macht es jedoch wenig Sinn eine View zu verwenden, nur um irgendwelche Properties zu setzen...

Ausserdem macht render Extension namentlich nicht besonders viel Sinn wenn diese nicht gerendert wird. Deshalb haben wir uns für eine andere Lösung entschieden :+1: